component/src/main/java/org/wso2/extension/siddhi/store/influxdb/InfluxDBStore.java

```diff
@@ -0,0 +1,565 @@
+        String condition = influxCompiledCondition.getCompiledQuery();
+        Map<Integer, Object> constantMap = influxCompiledCondition.getParametersConstants();
+
+        try {
+            QueryResult queryResult = this.getSelectQueryResult(containsConditionParameterMap, condition, constantMap);
+
+            if (queryResult.getResults().get(0).getSeries() == null) {
+                return false;
+            } else {
+                return true;
+            }
+
+        } catch (InfluxDBException e) {
+            throw new InfluxDBTableException("Error performing 'contains'  on the table : '" + this.tableName + "': "
+                    + e.getMessage(), e);
+        }
+    }
+
+    /**
+     * Delete all matching records.
+     *
+     * @param deleteConditionParameterMaps map of matching StreamVariable Ids and their values corresponding to the
+     *                                     compiled condition
+     * @param compiledCondition            the compiledCondition against which records should be matched for deletion
+     **/
+    @Override
+    protected void delete(List<Map<String, Object>> deleteConditionParameterMaps, CompiledCondition compiledCondition) {
+
+        InfluxDBCompiledCondition influxCompiledCondition = (InfluxDBCompiledCondition) compiledCondition;
+        String condition = influxCompiledCondition.getCompiledQuery();
+        Map<Integer, Object> constantMap = influxCompiledCondition.getParametersConstants();
+        String findCondition;
+
+        influxdb.setDatabase(database);
+        Query query;
+
+        try {
+            StringBuilder lines = new StringBuilder();
+            lines.append(DELETE_QUERY).append(this.tableName).append(INFLUXQL_WHERE);
+            findCondition = lines.toString();
+
+            for (Map<String, Object> map : deleteConditionParameterMaps) {
+                for (Map.Entry<String, Object> entry : map.entrySet()) {
+                    condition = condition.replaceFirst(Pattern.quote("?"), entry.getValue().toString());
+                }
+            }
+            for (Map.Entry<Integer, Object> entry : constantMap.entrySet()) {
+                Object myObject = entry.getValue();
+                condition = condition.replaceFirst(Pattern.quote("*"), myObject.toString());
+            }
+            findCondition = findCondition + condition;
+
+            query = new Query(findCondition, database);
+            influxdb.query(query);
+
+        } catch (InfluxDBException e) {
+            throw new InfluxDBTableException("Error deleting records from table '" + this.tableName + "': "
+                    + e.getMessage(), e);
+        }
+    }
+
+    @Override
+    protected void update(CompiledCondition compiledCondition, List<Map<String, Object>> list,
+                          Map<String, CompiledExpression> map, List<Map<String, Object>> list1)
+            throws ConnectionUnavailableException {
+
```

Comment:
Update operations are not supported? What happens if a user defines it? Is there any exceptions thrown?

Reply:
https://github.com/wso2-extensions/siddhi-store-influxdb/pull/1/commits/520b093086a08e2d8d9df4297dcd71914e15813b

Reply:
https://github.com/wso2-extensions/siddhi-store-influxdb/pull/1/commits/520b093086a08e2d8d9df4297dcd71914e15813b